XenAdmin/SettingsPanels/NetworkOptionsEditPage.Designer.cs

```diff
@@ -0,0 +1,88 @@
+        {
+            if (disposing && (components != null))
+            {
+                components.Dispose();
+            }
+            base.Dispose(disposing);
+        }
+
+        #region Component Designer generated code
+
+        /// <summary> 
+        /// Required method for Designer support - do not modify 
+        /// the contents of this method with the code editor.
+        /// </summary>
+        private void InitializeComponent()
+        {
+            System.ComponentModel.ComponentResourceManager resources = new System.ComponentModel.ComponentResourceManager(typeof(NetworkOptionsEditPage));
+            this.radioButtonDisable = new System.Windows.Forms.RadioButton();
+            this.radioButtonEnable = new System.Windows.Forms.RadioButton();
+            this.labelRubric = new System.Windows.Forms.Label();
+            this.tableLayoutPanel1 = new System.Windows.Forms.TableLayoutPanel();
+            this.tableLayoutPanel1.SuspendLayout();
+            this.SuspendLayout();
+            // 
+            // radioButtonDisable
+            // 
+            resources.ApplyResources(this.radioButtonDisable, "radioButtonDisable");
+            this.radioButtonDisable.Name = "radioButtonDisable";
+            this.radioButtonDisable.TabStop = true;
+            this.radioButtonDisable.UseVisualStyleBackColor = true;
+            // 
+            // radioButtonEnable
+            // 
+            resources.ApplyResources(this.radioButtonEnable, "radioButtonEnable");
+            this.radioButtonEnable.Name = "radioButtonEnable";
+            this.radioButtonEnable.TabStop = true;
+            this.radioButtonEnable.UseVisualStyleBackColor = true;
+            // 
+            // labelRubric
+            // 
+            resources.ApplyResources(this.labelRubric, "labelRubric");
+            this.labelRubric.BackColor = System.Drawing.Color.Transparent;
+            this.labelRubric.Name = "labelRubric";
+            // 
+            // tableLayoutPanel1
+            // 
+            resources.ApplyResources(this.tableLayoutPanel1, "tableLayoutPanel1");
+            this.tableLayoutPanel1.Controls.Add(this.labelRubric, 0, 0);
+            this.tableLayoutPanel1.Name = "tableLayoutPanel1";
+            // 
+            // NetworkOptionsEditPage
+            // 
+            resources.ApplyResources(this, "$this");
+            this.AutoScaleMode = System.Windows.Forms.AutoScaleMode.Dpi;
+            this.Controls.Add(this.tableLayoutPanel1);
+            this.Controls.Add(this.radioButtonDisable);
+            this.Controls.Add(this.radioButtonEnable);
+            this.Name = "NetworkOptionsEditPage";
+            this.tableLayoutPanel1.ResumeLayout(false);
+            this.tableLayoutPanel1.PerformLayout();
+            this.ResumeLayout(false);
+            this.PerformLayout();
+
+        }
+
+        #endregion
+
+        private System.Windows.Forms.RadioButton radioButtonDisable;

```

Comment:
Having now seen the design, I agree with Tina that this should be a checkbox not radio buttons. But I also don't want a tab with just one checkbox on it. Is there anywhere else it can fit?